test-code/recode_road_class.R

```diff
@@ -0,0 +1,50 @@
+# Date      27/06/2022
```

Comment:
I suggest moving this file into the R directory.

Reply:
Which directory is this? 
`openinfra/code` ?